Wabbajack.Installer/AInstaller.cs

```diff
@@ -370,29 +370,28 @@ public async Task DownloadMissingArchives(List<Archive> missing, CancellationTok
                 UpdateProgress(1);
             }
         }
-        
-        await missing
-            .Shuffle()
-            .Where(a => a.State is not Manual)
-            .PDoAll(async archive =>
-            {
-                _logger.LogInformation("Downloading {Archive}", archive.Name);
-                var outputPath = _configuration.Downloads.Combine(archive.Name);
-                var downloadPackagePath = outputPath.WithExtension(Ext.DownloadPackage);
 
-                if (download)
-                    if (outputPath.FileExists() && !downloadPackagePath.FileExists())
-                    {
-                        var origName = Path.GetFileNameWithoutExtension(archive.Name);
-                        var ext = Path.GetExtension(archive.Name);
-                        var uniqueKey = archive.State.PrimaryKeyString.StringSha256Hex();
-                        outputPath = _configuration.Downloads.Combine(origName + "_" + uniqueKey + "_" + ext);
-                        outputPath.Delete();
-                    }
+        var missingBatches = missing.Batch(100).ToList();
 
-                var hash = await DownloadArchive(archive, download, token, outputPath);
-                UpdateProgress(1);
-            });
+        List<Task> downloadBatches = [];
```

Comment:
I think the PDoAllBatched method is more suited for batching here

Reply:
That shouldn't be needed anymore. The 10 min cooldown Nexus kept hitting me with seems to have been from the repeated requests to the validate endpoint, which has been fixed. 

Will revert this change.

Reply:
It's under the Debug menu at the top > Windows > Output